wsi_deid/import_export.py

```diff
@@ -110,9 +110,32 @@ def getSchema():
     # that contain a single file that parses as valid json, then we need to
     # load those files and combine them into a single schema in lieu of using
     # this default SCHEMA_FILE_PATH
-    logger.info('Using %s (length %d) for schema' % (
-        SCHEMA_FILE_PATH, os.path.getsize(SCHEMA_FILE_PATH)))
-    return json.load(open(SCHEMA_FILE_PATH))
+    schemaFolderId = Setting().get(PluginSettings.WSI_DEID_SCHEMA_FOLDER)
+    mergedSchema = {}
+
+    if schemaFolderId:
+        schemaFolder = Folder().load(schemaFolderId, force=True)
```

Comment:
We'll want to check if the schemaFolder is non-None, since if the folder was somehow deleted, this will be None.

Reply:
added in 2a93feaf7fb521edcaeb54d9812d4320d10df69e